packages/babel-types/src/traverse/traverseFast.ts

```diff
@@ -7,28 +7,39 @@ import type * as t from "../index.ts";
  */
 export default function traverseFast<Options = object>(
   node: t.Node | null | undefined,
-  enter: (node: t.Node, opts?: Options) => void,
+  enter: (node: t.Node, opts?: Options) => void | "skip" | "stop",
   opts?: Options,
-): void {
-  if (!node) return;
+): boolean {
+  if (!node) return false;
```

Comment:
Please document what the `true`/`false` return types mean, in the JSDoc comment.